src/backend/app.js

```diff
@@ -20,6 +20,89 @@ app.use(express.json());
 app.use(cors());
 
 router.use("/meals", mealsRouter);
+router.get('/', (req, res) => {
+  res.send('Hi friend');
+});
+//Respond with all meals in the future
+//(relative to the when datetime):
+router.get('/future-meals', async (req, res) => {
+  try {
+    //const NOW = new Date();
+    const futureMeals = await knex.raw(
+      `
+  SELECT * FROM meals
+   WHERE date > NOW()`
+    );
+    res.json(futureMeals);
+  } catch (error) {
+    throw error;
+  }
+});
+//Respond with all meals in the past
+//(relative to the when datetime):
+router.get('/past-meals', async (req, res) => {
+  try {
+    //const now = new Date();
+    const pastMeals = await knex.raw(
+      `
+  SELECT * FROM meals
+   WHERE date < NOW()`
+    );
+
+    router.use('/meals', mealsRouter);
```

Comment:
Why did you put it in here?

Reply:
could you explain more about what do you mean :)